src/User_Application_Password_Command.php

```diff
@@ -197,8 +198,13 @@ static function ( $application_password ) use ( $field, $value ) {
 			$fields = explode( ',', $assoc_args['fields'] );
 		}
 
-		$formatter = new Formatter( $assoc_args, $fields );
-		$formatter->display_items( $application_passwords );
+		if ( ! empty( $assoc_args['format'] ) && 'ids' === $assoc_args['format'] ) {
+			$formatter = new Formatter( $assoc_args, $fields );
+			$formatter->display_items( wp_list_pluck( $application_passwords, 'uuid' ) );
+		} else {
+			$formatter = new Formatter( $assoc_args, $fields );
+			$formatter->display_items( $application_passwords );
+		}
```

Comment:
Nit: move `new Formatter()` up before the condition:

```suggestion
		$formatter = new Formatter( $assoc_args, $fields );

		if ( ! empty( $assoc_args['format'] ) && 'ids' === $assoc_args['format'] ) {
			$formatter->display_items( wp_list_pluck( $application_passwords, 'uuid' ) );
		} else {
			$formatter->display_items( $application_passwords );
		}
```

Aside: I noticed many commands just do `echo implode( ' ', wp_list_pluck( ... ) );` instead 🤔  Not sure why. Using the formatter is cleaner.

Reply:
I agree. Those codes are from 7 or 8 years ago. Earlier such approach was used to print ids.

Reply:
The change you mentioned did not work. So I have kept as it is.

Reply:
How does moving `$formatter = new Formatter( $assoc_args, $fields );` up not work? 🤔 
It just removes a duplicated line by moving it out of the conditional block.

Reply:
In Formatter value is passed by reference. So it messes up the actual value.